docs/docs/modeling.md

```diff
@@ -108,6 +107,25 @@ classDiagram
 
 ## Category - カテゴリー
```

Comment:
You might need to mention the minimal number of `Category` too as well as the maximum you mentioned below.